turbopack/crates/turbopack-core/src/chunk/mod.rs

```diff
@@ -713,6 +713,18 @@ pub trait ChunkType: ValueToString {
     ) -> Vc<usize>;
 }
 
+pub fn round_chunk_item_size(size: usize) -> usize {
+    if size == 0 {
+        return 0;
+    }
+    let a = size.next_power_of_two();
+    if a == size {
+        size
+    } else {
+        size & (a | (a >> 1) | (a >> 2))
```

Comment:
So the idea here is to have exponentially growing spacing between sizes (just like `next_power_of_two`), but just slightly smaller spacing?

Reply:
Yep. `next_power_of_two` would have a new size every time the value doubles.
This implementation has a new size every time the value increases by 50%